Source/WebCore/bindings/scripts/CodeGeneratorJS.pm

```diff
@@ -5545,7 +5545,7 @@ sub GenerateAttributeGetterTrampolineDefinition
         push(@templateParameters, "CastedThisErrorBehavior::Assert");
     }
     
-    push(@$outputArray, "JSC_DEFINE_CUSTOM_GETTER(${attributeGetterName}, (JSGlobalObject* lexicalGlobalObject, EncodedJSValue thisValue, PropertyName attributeName))\n");
+    push(@$outputArray, "REFERENCED_FROM_ASM JSC_DEFINE_CUSTOM_GETTER(${attributeGetterName}, (JSGlobalObject* lexicalGlobalObject, EncodedJSValue thisValue, PropertyName attributeName))\n");
```

Comment:
We don't need this change because JSC_DECLARE_CUSTOM_GETTER already contains REFERENCED_FROM_ASM.

Reply:
Yeah you're right - the code compiles without this change. I must be missing something though because I can't see where `REFERENCED_FROM_ASM` is added for `JSC_DEFINE_CUSTOM_GETTER` (`DEFINE`, not `DECLARE`).

`JSC_DECLARE_CUSTOM_GETTER` contains `REFERENCED_FROM_ASM` via `JSC_DECLARE_JIT_OPERATION_WITHOUT_WTF_INTERNAL`.

`JSC_DEFINE_CUSTOM_GETTER` uses `JSC_DEFINE_JIT_OPERATION_WITHOUT_VARIABLE` which doesn't contain `REFERENCED_FROM_ASM`.

Reply:
Right. it's enough if the declaration has the attribute.